docs/Installation.md

```diff
@@ -1,114 +1,221 @@
+- [Access the management container](#management-container)
+- [Install the operating systems on the nodes](#os-nodes)
+- [Setup the storage nodes](#storage-nodes)
+- [Install JumpScale services on nodes](#jumpscale-nodes)
+- [Deploy virtual machine images](#deploy-images)
+
+
+<a id="prerequisites"></a>
+## Meet the prerequisites
+
+- 3 dedicated controller nodes
+- 10 dedicated CPU nodes
+- 4 dedicated storage nodes
+- SSH access credentials for all nodes
 - Swap needs to be off on each node
 - Each node needs to be able to access each other node in the cluster
-- three files for ssl verification need to be added to the  specified path in the yaml config on the machine all wit
-  the same name and have these extensions:
+- Three files for SSL verification need to be added to the specified path in the YAML config on the machine all with the same name and have these extensions:
   - .csr
   - .crt
   - .key
 - `jsonschema` python library
 
-## Full environment setup
 
-Using the docker image `openvcloud/js9` create a docker container which will have a JumpScale 9 installation.
+<a id="create-config"></a>
+## Create the configuration file
+
+Installing an OpenvCloud cluster is done based on a single `system-config.yaml` configuration file.
+
+This configuration files describes:
+- Switch configuration
+- Operating system installations
+- Kubernetes cluster running on the controllers, which hosts the OpenvCloud master and and all other controller components
+- CPU and storage nodes installation
+- ...
+
+The `system-config.yaml` file needs to be stored and maintained in the root of a Git repository on `https://docs.greenitglobe.com`; for each G8 installation there is distinct Git repository on https://docs.greenitglobe.com
+
+The following rules apply:
+- The name of the repository should be formatted `env_<<descriptive environment specification>>`, e.g. `env_be-g8-4`
+- The repository needs to be put in an organization that represents the partner or customer, e.g. `gigtech` organization for G8s owned by GIG itself, or `digitalenergy` for our Russian partner
+
+Some example repositories:
+  - https://docs.greenitglobe.com/gigtech/env_be-g8-4
+  - https://docs.greenitglobe.com/gigtech/env_se-sto-en01-001
+  - https://docs.greenitglobe.com/digitalenergy/env_mr4
+
+An example of a `system-config.yaml` can be found here: https://github.com/0-complexity/openvcloud_installer/blob/master/scripts/kubernetes/config/system-config.yaml
+
+> **Important** A common technique to create a `system-config.yaml` is to make a copy from another environment and start editing. Please make sure to alter the `ssh.private-key` setting, and not just leave the copy from the other environment.
+
+
+<a id="validate-config"></a>
+## Validate the configuration file
+
+Having valid configuration is off course very important. Validating is done with the OpenvCloud environment manager, a.k.a. **Menaja** (Swahili for 'manager'), available on https://menaja.gig.tech.
```

Comment:
with stupid names come stupid typos and ofcourse nobody can remember this anyway its https://meneja.gig.tech/ not menaja

Reply:
corrected